helper/stringhelper/stringhelper.go

```diff
@@ -0,0 +1,16 @@
+package stringhelper
+
+import "strings"
+
+// IsContainedInSlice checks if the given string is contained in the slice.
+// If case insensitive is enabled, checks are case insensitive.
+func IsContainedInSlice(s []string, c string, caseIns bool) bool {
```

Comment:
This is never called with caseIns = false. :)

Reply:
Not yet. I just want to make sure that its there when someone needs it 😃